### PR TITLE
Added `mosaicml-streaming` to known list

### PR DESCRIPTION
**Pull request by pritishpai**
## Changes
Added `mosaicml-streaming` to known list 

### Linked issues
Partly resolve #1931

### CODE REVIEW

**Review by nfx — CHANGES_REQUESTED** [review]
reduce the size of the diff to <20 lines

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -32580,6 +32591,71 @@
     "functorch": [],
     "torch": []
   },
+  "torchvision": {
+    "torchvision": [],
+    "torchvision._internally_replaced_utils": [],
+    "torchvision._meta_registrations": [],
+    "torchvision._utils": [],
+    "torchvision.datasets": [],
```
Comment (pritishpai) [inline src/databricks/labs/ucx/source_code/known.json:32599]: Had to keep torchvision.datasets as one of the libraries has to be excluded, so just included all but torchvision.datasets.widerface
`    "torchvision.datasets.widerface": [
      {
        "code": "direct-filesystem-access",
        "message": "The use of direct filesystem references is deprecated: /"
      }
    ],`

**Review by nfx — APPROVED** [review]
lgtm

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**